Pushover (#1221)
n8n-io/n8n-docs: docs/integrations/builtin/app-nodes/n8n-nodes-base.pushover.md

# Pushover

> The Pushover node allows you to automate work in Pushover, and integrate Pushover with other applications. n8n has built-in support for a wide range of Pushover features, including pushing messages. 

Reviewer: ```suggestion
The Pushover node allows you to automate work in Pushover, and integrate Pushover with other applications. n8n supports sending push notifications with Pushover.
```
